string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,28 @@
+/* exported titleCase */
+function titleCase(title) {
+  title = title.split(' ');
+  const newTitle = [];
+  let temp = null;
+  for (let i = 0; i < title.length; i++) {
+    temp = title[i];
+    if (temp.length > 3 || title[i].charAt(0) === 'w') {
```

Comment:
This part with the hard-coded `"w"` is a no-go. Try reworking your solution to store a list of short words and check if a given word is in that list. Your solution should be able to handle _any_ title and not just the set provided in the tests.